packages/site/src/App.tsx

```diff
@@ -4,9 +4,13 @@ import { NitroRpcClient } from "@statechannels/nitro-rpc-client";
 import { NetworkBalance } from "./components/NetworkBalance";
 import "./App.css";
 import TopBar from "./components/TopBar";
+import { QUERY_KEY } from "./constants";
 
 function App() {
-  const [url, setUrl] = useState("localhost:4005");
+  const [url, _] = useState(
```

Comment:
I don't think we need the `useState` hook.

Reply:
Agreed.